homeassistant/components/modbus/__init__.py

```diff
@@ -239,7 +250,7 @@
             vol.Optional(CONF_HVAC_MODE_REGISTER): vol.Maybe(
                 {
                     CONF_ADDRESS: cv.positive_int,
-                    CONF_HVAC_MODE_VALUES: {
+                    CONF_MODE_VALUES: {
```

Comment:
Happy to address this, but just to double-check my understanding of what is and isn't allowed. Since I've not changed the user-facing configuration keyword nested under `hvac_mode_register` (which remains `value`), I thought this was permissible and not really a user-facing deprecation.

I could create a second static `CONF_FAN_MODE_VALUES = "value"` to keep them both independent and leave the `CONF_HVAC_MODE_VALUES` static class variable name untouched?